docs/platform-support.md

```diff
@@ -64,6 +64,7 @@ Tier 3 platforms are currently:
 | linux/386     | [@andrzej-stencel](https://github.com/andrzej-stencel)                                                                                                         |
 | linux/arm     | [@Wal8800](https://github.com/Wal8800), [@atoulme](https://github.com/atoulme)                                                                                 |
 | linux/ppc64le | [@IBM-Currency-Helper](https://github.com/IBM-Currency-Helper), [@adilhusain-s](https://github.com/adilhusain-s), [@seth-priya](https://github.com/seth-priya) |
+| linux/riscv64 | [@shanduur](https://github.com/shanduur)                                                                                                                       |
```

Comment:
@shanduur you might like to first apply to become an OpenTelemetry member - see https://github.com/open-telemetry/community/issues/new?template=membership.md

Reply:
Nothing specific - completely agree that the contributions you make towards riscv64 are enough, and I'd like to sponsor you if you'd have me.